content/api/resources/liquiditypools/operations.mdx

```diff
@@ -0,0 +1,174 @@
+---
+title: Retrieve related Operations
+order: 50
+---
+
+import { Endpoint } from "components/Endpoint";
+import { ExampleResponse } from "components/ExampleResponse";
+import { CodeExample } from "components/CodeExample";
+import { AttributeTable } from "components/AttributeTable";
+
+This endpoint represents successful operations referencing a given liquidity pool and can be used in streaming mode.
+
+Streaming mode allows you to listen for new operations referencing this liquidity pool as they are added to the Stellar ledger. If called in streaming mode, Horizon will start at the earliest known operation unless a `cursor` is set, in which case it will start from that `cursor`. By setting the `cursor` value to `now`, you can stream operations created since your request time.
+
+<Endpoint>
+
+
+|  |  |
+| --- | --- |
+| GET | /liquidity_pools/:liquidity_pool_id/operations?cursor={paging_token}&order={asc,desc}&limit={1-200}&include_failed{true,false}&join={transactions} |
+
+</Endpoint>
+
+
+<AttributeTable>
+
+
+- ARGUMENT
+  - REQUIRED
+  - DESCRIPTION
+- liquidity_pool_id
+  - required
+  - This liquidity pool's id encoded in a hex string representation.
+- cursor
+  - optional
+  - A number that points to a specific location in a collection of responses and is pulled from the `paging_token` value of a record.
+- order
+  - optional
+  - A designation of the order in which records should appear. Options include `asc`(ascending) or `desc` (descending). If this argument isn’t set, it defaults to `asc`.
+- limit
+  - optional
+  - The total number of records returned. The limit can range from 1 to 200 - an upper limit that is hardcoded in Horizon for performance reasons. If this argument isn’t designated, it defaults to 10.
+- include_failed
+  - optional
+  - Set to `true` to include failed operations in results. If this argument isn't set, it defaults to `false`.
+- join
+  - optional
+  - Set to `transactions` to include the transactions which created each of the operations in the response.
+
+</AttributeTable>
+
+
+<CodeExample title="Example Request">
+
+
+```curl
+curl "https://horizon.stellar.org/liquidity_pools/67260c4c1807b262ff851b0a3fe141194936bb0215b2f77447f1df11998eabb9/operations?limit=3"
+```
+
+```js
+var StellarSdk = require("stellar-sdk");
+var server = new StellarSdk.Server("https://horizon.stellar.org");
+
+server
+  .operations()
+  .forLiquidityPool(
+    "67260c4c1807b262ff851b0a3fe141194936bb0215b2f77447f1df11998eabb9",
+  )
+  .call()
```

Comment:
Maybe we add `.limit(3)` to match the curl request? (or drop that parameter from that example)